docs/content/getting-started/releases.mdx

```diff
@@ -33,7 +33,9 @@ Like non-deprecated public stable APIs, deprecated public stable APIs will not b
 
 ## Dagster integration libraries
 
-Dagster's integration libraries are in the 0.y.z of [Semantic Versioning](https://semver.org/). While technically this is "anything goes", we are conservative about making changes and provide guidance about when to expect breaking changes:
+Dagster’s integration libraries haven’t yet achieved the same API maturity as Dagster core. For this reason, integration libraries remain on a pre-1.0 versioning track (in general 0.y.z of [Semantic Versioning](https://semver.org/); and 0.16+ as of Dagster 1.0.0) for the time being. However, 0.16+ library releases remain fully compatible with Dagster 1.x. In the coming months, we will graduate integration libraries one-by-one to the 1.x versioning track as they achieve API maturity.
```

Comment:
I'd remove the `in the coming months piece`.

Reply:
good call